heron/packing/tests/java/org/apache/heron/packing/roundrobin/RoundRobinPackingTest.java

```diff
@@ -255,6 +282,41 @@ public void testCompleteRamMapRequested() throws Exception {
     Assert.assertEquals(totalInstances, packingPlanExplicitRamMap.getInstanceCount());
   }
 
+  /**
+   * Test the scenario CPU map config is completely set
+   */
+  @Test
+  public void testCompleteCpuMapRequested() throws Exception {
```

Comment:
Can we add these cases:
- container cpu is more than needed by instances.
- container cpu is less than needed by instances.

also, other processes in container could need a fixed amount of CPU as well. So if container cpu == needed by instances, it might be better to return false.

Reply:
👍 Added more tests to cover the mentioned scenarios